script/brave_license_helper.py

```diff
@@ -38,6 +38,34 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
         os.path.join('brave', 'third_party', 'libaddressinput'),
         os.path.join('brave', 'patches', 'third_party'),
 
+        # Duplicate notices already included in brave-core notices
+        os.path.join('brave', 'vendor', 'omaha', 'omaha', 'third_party',
+                     'chrome'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'libzip'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'lzma'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'zlib'),
+
+        # Dependencies already included in the main breakpad notice.
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'breakpad',
+                     'src', 'third_party', 'curl'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'breakpad',
+                     'src', 'third_party', 'libdisasm'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'breakpad',
+                     'src', 'third_party', 'linux'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'breakpad',
+                     'src', 'third_party', 'mac_headers'),
+
+        # Dependencies which are no longer present in the codebase. This should
+        # be cleaned up in the upstream dependency including this.
+        os.path.join('brave', 'vendor', 'omaha', 'omaha', 'third_party',
+                     'hashlib'),
+
+        # No licensing information in recursive dependency. This should be fixed
+        # by the upstream dependency including this.
+        os.path.join('brave', 'vendor', 'omaha', 'omaha', 'third_party', 'bar'),
```

Comment:
The `bar/` directory does not exist for me. Was this line added by accident?

Reply:
You're right, it looks like I hadn't moved my local repo to the new `main` branch (I was still on `master`). That's where `bar` was. I'll take it out.